common/adminBlock/adminBlockHeader.go

```diff
@@ -18,10 +18,10 @@ import (
 	"github.com/FactomProject/factomd/common/primitives"
 )
 
-// Admin Block Header
+// ABlockHeader contains header information for the Admin Block
 type ABlockHeader struct {
 	PrevBackRefHash interfaces.IHash `json:"prevbackrefhash"`
-	DBHeight        uint32           `json:"dbheight"`
+	DBHeight        uint32           `json:"dbheight"` // The directory block height where this admin block is located
 
```

Comment:
remove space so the json tags are easier to read (go fmt will line them up)

Reply:
I'm not sure I understand this comment, which space should I remove? My IDE is automatically lining them up like that.

Reply:
Nevermind, i figured it out based on your other comment below. I removed the new line between structures so the jsons were aligned.